cub/device/dispatch/dispatch_radix_sort.cuh

```diff
@@ -1116,6 +1123,10 @@ struct DispatchRadixSort :
                 pass_config.even_share.grid_size, pass_config.upsweep_config.block_threads, (long long) stream,
                 pass_config.upsweep_config.items_per_thread, pass_config.upsweep_config.sm_occupancy, current_bit, pass_bits);
 
+            // Zero the spine.
+            //if (CubDebug(error = cudaMemsetAsync(d_spine, 0, sizeof(OffsetT) * spine_length, stream))) break;
```

Comment:
Was this intentional? We should uncomment or remove these lines.

Reply:
This was something I tried to do. Feel free to remove it.